package.json

```diff
@@ -13,8 +13,10 @@
     "@types/node": "^13.9.2",
     "browserify": "^16.5.0",
     "gulp": "^4.0.2",
+    "gulp-open": "^3.0.1",
```

Comment:
it's not needed

Reply:
Actually, the ordinary open does not work on my machine, and I can't seem to debug it. I have a reasonably standard Ubuntu setup, so I think others might also face the issue.

Reply:
Have you already tried again after my review? I'm using ubuntu 18.04, and it's working quite well for me.

By the way, the browser is still not reloading when a file is modified. You'll need to fix that :confused: 

Thanks for contributing.

Reply:
Ok the livereload is not working because open() is opening the file rather than the localhost server. So I can't use gulp-open anyway. Once I remove that the code works, and live reload works too. 
However, I have tried open as a field in webserver() and it still does not work. I have Kubuntu 20.04.
Debugging seems to point the error at the old version of the npm package open. gulp-webserver uses version 0.0.5 which is 7 years old, and not updated for newer KDE I guess. Putting open as a dependency in the main package.json did not help either, as I guess it does not play well with the TS definitions in the new version.
As your version opens for some people, and reloads for all, will you merge it? Better than open for none, I feel. If yes, I will commit the new changes

Reply:
It would be easier for me to understand if you show some code. Is gulp watch launching the server in the browser and reloading correctly? When I started the project, I was using a Manjaro KDE, and I don't remember any problem with Gulp.

Reply:
```
function watch() {
    gulp.watch('./src/**/*.ts', gulp.series(copyHTML, bundle));
    gulp.watch('./public/**/*.html', gulp.series(copyHTML, bundle));
    gulp.watch('./public/**/*.scss', style);
}

function liveReload() {
    gulp.src('dist/').pipe(webserver({
        livereload: true,
        open: 'index.html'
    }));
}

...
exports.watch = gulp.series(copyHTML, style, bundle, copyImages, gulp.parallel(wt, live));
```
Here is the relevant code.
When run, it gives this output
```
[19:56:54] Finished 'copyImages' after 51 ms
[19:56:54] Starting 'wt'...
[19:56:54] Starting 'live'...
[19:56:54] Webserver started at http://localhost:8000
```
and stops there without opening automatically. 
I can manually open the URL and it works, reloads automatically etc. But just does not open automatically.

Reply:
I am not facing any problem with gulp, just with gulp-webserver, and only related to opening automatically. Live reload is now working in the code shown above

Reply:
Ok, commit that and I'll merge it. This isn't a huge problem.